source/_integrations/calendar.markdown

```diff
@@ -54,6 +62,79 @@ automation:
         message: >-
           Event {{ trigger.calendar_event.summary }} @
           {{ trigger.calendar_event.start }}
+  mode: parallel
+```
+{% endraw %}
+
+{% enddetails %}
+
+{% details "Example: Calendar Event Light Schedule " %}
+
+This example consists of:
+- For the calendar entity ` calendar.device_automation`
+- When event summary contains `Front Lights`
+- Turn on and off light named `light.front` when the event starts and ends
+- Only runs for one event at a time
+
+{% raw %}
+```yaml
+automation:
+  alias: Front Light Schedule
+  trigger:
+    - platform: calendar
+      event: start
+      entity_id: calendar.device_automation
+    - platform: calendar
+      event: end
+      entity_id: calendar.device_automation
+  condition:
+  - condition: template
+    value_template: "{{ 'Front Lights' in trigger.calendar_event.summary }}"
+  action:
+    service: >
+      {% if trigger.event == "start" %}
+        light.turn_on
+      {% else %}
+        light.turn_off
+      {% endif %}
+    target: light.front
   mode: single
 ```
 {% endraw %}
+
+{% enddetails %}
+
+{% details "Example: Calendar Event Binary Sensor Template" %}
```

Comment:
I'm not sure if this is an example we should document in our end-user documentation. It is pretty advanced and very specific.

Reply:
Yes, understandable. Maybe I can tell you about the motivation, then we can figure out the right way to approach it:

I think it makes sense to start trying to phase out how people currently use the state in the calendar entity.  For example, folks use `google_calendars.yaml` which includes things like creating multiple entities for a single calendar, or allowing search terms on top of events.    It is also simple to use a calendar entity as a condition today which may involve checking the current state of the calendar event and i don't have an alternative answer at the moment. Users generally seems to *really* love binary sensors, and I think they may need help finding alternatives for various use cases.

(I myself am not an advanced automation user as you can see from my attempts to write automations above, so still working to help fill in the gaps for myself too)

Reply:
I think i may have convinced myself to instead just spend more time thinking about calendar conditions.. 👍🏼